src/app/pages/MediaArticlePage/MediaArticlePage.styles.ts

```diff
@@ -27,6 +27,14 @@ export default {
         gridColumn: '1 / span 8',
       },
     }),
+  cafPrimaryColumn: ({ mq }: Theme) =>
```

Comment:
yeah that makes sense as caf is more of a dev flag ! With the media query right now ive changed it so instead of making it full width it just starts at the 3rd column out of 8 which centered it, but full width at all breakpoints makes more sense - will double check with UX as well.

Reply:
Thanks @emilysaffron. One for UX maybe, but do we need to constraint the width on desktop to a certain width? The current MAP pages are centred but don't take up the entire width, looks like they are around `45rem` give or take.

Reply:
Wendy has signed off on it being left as is full width 😊